.github/workflows/makefile.yml

```diff
@@ -15,31 +19,59 @@ jobs:
     - name: shellcheck
       run: make -C build shellcheck
 
-  build:
+  build_containers:
     needs: shellcheck
     runs-on: ubuntu-latest
     strategy:
       matrix:
         os: [ubuntu2004, fedora33]
+        platform: [linux/amd64]
```

Comment:
Since you have dockerx working now, can we add back in `linux/arm64` here as well?

Reply:
@mestery No, first the Dockerhub or Guthub Container Registry needs to be enabled (and some script code in the ipdk.sh build function  needs to be adapted) because the standard local docker image store doesn't support multi arch images 😔 Buildx needs to push directly to one of those remote registries. One solution I found was talking about a local registry as action service, but I didn't have enough time today to test that. Will try that on Monday...

Reply:
OK. I pushed a change to try this out before you had replied, feel free to force push back over top of that whenever you want to get rid of it.

Reply:
I think I solved creating the multiarch problem. But now we need to solve the crashing linux/arm64 build

Reply:
We can add `linux/arm64` to the platform environment variable when the pr's for arm64 support are pushed I think. Also it would be nice if we can add a native arm64 builder host to buildx but that probably depends on your endeavor getting oci access....